n2v/grid/grider.py

```diff
@@ -19,22 +19,24 @@
 
 from .cubeprop import Cubeprop
 
-default_grid = np.concatenate((np.linspace(-10,10,20, dtype=np.half)[:,None],
+_default_grid = np.concatenate((np.linspace(-10,10,20, dtype=np.half)[:,None],
                               np.linspace(-10,10,20, dtype=np.half)[:,None],
                               np.linspace(-10,10,20, dtype=np.half)[:,None]), axis=1)
 
 class Grider(Cubeprop):
 
-    def grid_to_blocks(self, grid):
+    def grid_to_blocks(self, grids, basis=None):
         """
-        Generate list of blocks to allocate given grid
+        Generate list of blocks to allocate given grids
```

Comment:
I don't understand why the "grid" was changed to "grids." Are there any multiple grids given at a time? Also,  we save one keystroke.